services/app-web/src/pages/StateSubmission/Contacts/Contacts.tsx

```diff
@@ -491,173 +470,6 @@ const Contacts = ({
                                         </FieldArray>
                                     </fieldset>
                                 </SectionCard>
-
```

Comment:
Asked a question about this in Slack. The way this is implemented right now we would just be disappearing the additional actuary fields from the "flag off" state for LINK_ RATES.

That would be a regression on production, where LINK_RATES is not turned on. I wonder if this is related to the cypress issues you are seeing (not knowing what they are)